lib/pharos/addons/cert-manager/addon.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+Pharos.addon 'cert-manager' do
+  version '0.2.3'
+  license 'Apache License 2.0'
+
+  issuer = custom_type {
```

Comment:
With newer version of `dry-struct` (?) this would just be a nested struct: http://dry-rb.org/gems/dry-struct/nested-structs/

```ruby
config do
  attribute :issuer do
    attribute :name, Pharos::Types::String
    ...
  end
end
```

Reply:
Yes, it is much better.